tests/e2e/v3_curl_test.go

```diff
@@ -385,6 +386,107 @@ func testV3CurlMaintenanceAlarmMissiongAlarm(cx ctlCtx) {
 	}
 }
 
+func TestV3CurlMaintenanceStatus(t *testing.T) {
+	testCtl(t, testV3CurlMaintenanceStatus, withCfg(*e2e.NewConfigNoTLS()))
+}
+
+func testV3CurlMaintenanceStatus(cx ctlCtx) {
+	if err := e2e.CURLPost(cx.epc, e2e.CURLReq{
+		Endpoint: "/v3/maintenance/status",
+		Value:    "{}",
+		Expected: expect.ExpectedResponse{
+			Value:         fmt.Sprintf(`.*"version":%q,"dbSize":.*,\s*"leader":.*,\s*"raftIndex":.*,\s*"raftTerm":.*,\s*"raftAppliedIndex":.*,\s*"dbSizeInUse":.*,\s*"storageVersion":.*`, version.Version),
+			IsRegularExpr: true,
+		},
+	}); err != nil {
+		cx.t.Fatalf("failed post maintenance status request: (%v)", err)
+	}
+}
+
+func TestV3CurlMaintenanceDefragment(t *testing.T) {
+	testCtl(t, testV3CurlMaintenanceDefragment, withCfg(*e2e.NewConfigNoTLS()))
+}
+
+func testV3CurlMaintenanceDefragment(cx ctlCtx) {
+	if err := e2e.CURLPost(cx.epc, e2e.CURLReq{
+		Endpoint: "/v3/maintenance/defragment",
+		Value:    "{}",
+		Expected: expect.ExpectedResponse{
+			Value: "{}",
```

Comment:
Just double checking this is intentional and there is no return we can validate here?

Reply:
It's intentional. The response is just "{}".
```
$ curl -L http://127.0.0.1:2379/v3/maintenance/defragment -X POST
{}
```